app/stac_api/models.py

```diff
@@ -36,22 +36,13 @@
 # instance so that it's not shared between all field instances.
 # HINT: Use a callable instead, e.g., use `list` instead of `[]`.""
 # <end quote>
```

Comment:
This comment (line 29-38) should be removed

Reply:
Done